lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -703,6 +703,9 @@ private function completeIdGeneratorMapping(ClassMetadataInfo $class)
 
             case ClassMetadata::GENERATOR_TYPE_CUSTOM:
                 $definition = $class->customGeneratorDefinition;
```

Comment:
It seems like we have a docblock issue here... could you please check if `customGeneratorDefinition` property has `|null`?

Reply:
27cfd3177c5410ae9fa07b2724e6030b99131e4d